docs/docs/dev_docs/tutorials/testing.md

```diff
@@ -80,6 +80,16 @@ Instead of creating new accounts in our test suite, we can use the ones already
 
 #include_code use-existing-wallets /yarn-project/end-to-end/src/guides/dapp_testing.test.ts typescript
 
+### Using debug options
+
+You can use the `debug` option in the `wait` method to get more information about the effects of the transaction. At the time of writing, this includes information about new note hashes added to the private data tree, new nullifiers, public data writes, new L2 to L1 messages, new contract information and newly visible notes.
```

Comment:
```suggestion
You can use the `debug` option in the `wait` method to get more information about the effects of the transaction. At the time of writing, this includes information about new note hashes added to the note hash tree, new nullifiers, public data writes, new L2 to L1 messages, new contract information and newly visible notes.
```

We've renamed the tree to note hash tree everywhere in the codebase a while ago.